subsys/net/ip/net_context.c

```diff
@@ -206,6 +216,14 @@ int net_context_get(sa_family_t family,
 			}
 		}
 
+#if defined(CONIFG_NET_TLS) || defined(CONFIG_NET_DTLS)
+		if (tls) {
```

Comment:
Maybe pedantic too, but looking at this code chunk, it's hard to say what type `tls` is (a pointer perhaps)? I'd suggest to rename that bool var as `is_tls`.